frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/TagEditorParam.styled.tsx

```diff
@@ -17,15 +17,14 @@ export const TagName = styled.h3`
 
 interface ContainerLabelProps {
   paddingTop?: boolean;
-  smallPaddingBottom?: boolean;
 }
 
 export const ContainerLabel = styled.div<ContainerLabelProps>`
   display: block;
-  font-weight: 700;
-  padding-bottom: ${props => (props.smallPaddingBottom ? "0.25rem" : "0.5rem")};
-  color: ${color("text-medium")};
+  margin-bottom: 0.5em;
   padding-top: ${props => (props.paddingTop ? "0.5rem" : "0")};
+  color: ${color("text-medium")};
```

Comment:
This a carried from the https://github.com/metabase/metabase/pull/38906#discussion_r1497003971 great suggestion from @kamilmielnik.